e2e/test/IoTHubServiceProxyE2ETests.cs

```diff
@@ -46,6 +46,27 @@ public async Task RegistryManager_AddAndRemoveDevice_WithProxy()
             await RegistryManager_AddDevice(httpTransportSettings).ConfigureAwait(false);
         }
 
+        [TestMethod]
```

Comment:
@varunpuranik  please create a separate file for RegistryManager tests (we didn't have this before). 
This file is for HTTP proxy tests only.

Reply:
Sure